Scripts/Player/Player.gd

```diff
@@ -1,45 +1,50 @@
 extends Node2D
 class_name PlayerNode
+## Represents the player character in the game world, handling movement, pathfinding,
```

Comment:
This is a menu PR, not sure why we're editing the player. The edits are good, just would like them moved to another PR.

Reply:
already in the main repo.

Reply:
sorry, i forgot to update this to merge with main, could you check again?